lib/charms/observability_libs/v1/cert_handler.py

```diff
@@ -278,7 +277,7 @@ def _on_certificate_available(self, event: CertificateAvailableEvent) -> None:
             content = {
                 "ca-cert": event.ca,
                 "server-cert": event.certificate,
-                "chain": json.dumps(event.chain),
+                "chain": event.chain_as_pem(),
```

Comment:
No need for backwards compat, because there was an omitted `json.loads`. I.e. the `chain` field is unused, otherwise we'd hear about it.